nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/AbstractProvenanceReporter.java

```diff
@@ -230,7 +230,7 @@ private void processProvenanceEvents(ReportingContext context) {
 
                 if (details != null && detailsAsError.contains(details.toLowerCase()))
                     source.put("status", "Error");
-                else if (httpCheck && Objects.equals(componentType, "InvokeHTTP"))
+                else if (httpCheck && Objects.equals(componentType, "InvokeHTTP") && "ATTRIBUTE_MODIFIED".equals(eventType != null ? eventType.toString() : null))
```

Comment:
- when I look in the `ProvenanceEventType` enum, it is called `ATTRIBUTES_MODIFIED`
- anyway, cleaner to compare on the enum (`ATTRIBUTES_MODIFIED.equals(eventType)`)
- no lines longer than 120 characters (you know what I think about horizontal scrolling :) )